.changeset/hip-maps-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": minor
```

Comment:
Rather than a `minor` change I would say its a patch one as we're changing an existing component

Reply:
Done [1c108c6](https://github.com/Blobscan/blobscan/pull/504/commits/1c108c6ea563872fee4b5f0b98792e3de307d3d6)